python/pyspark/ml/tests.py

```diff
@@ -807,6 +807,18 @@ def test_logistic_regression(self):
         except OSError:
             pass
 
+    def logistic_regression_check_thresholds(self):
+        self.assertIsInstance(
+            LogisticRegression(threshold=0.5, thresholds=[0.5, 0.5]),
+            LogisticRegressionModel
```

Comment:
@zero323 @yanboliang  Am I missing something?  Why should the LogisticRegression constructor return an instance of type LogisticRegressionModel?  Should you have to call fit?